configuration.yaml

```diff
@@ -37,12 +37,12 @@ nhead: 8
 lr: 0.0001
 max_epochs: 300
 batch_size: 2
-validation_split: 0.2 
+separate_split: 0.3 
```

Comment:
Probably a better name for the `separate_split` parameter is something like `validation_test_ratio`.